utils/python/wait_for_tests.py

```diff
@@ -11,6 +11,7 @@
 TEST_PENDING_STATUS       = "PEND"
 TEST_PASS_STATUS          = "PASS"
 TEST_FAIL_STATUS          = "FAIL"
+NO_BASELINE_STATUS    = "BFAIL"
```

Comment:
I may need to do some additional work in wait_for_tests to properly support this test status.